src/views/User/User.tsx

```diff
@@ -788,258 +872,548 @@ export class User extends React.PureComponent<UserProperties, UserState> {
+                                    <textarea
+                                        ref={(x) => (this.moderator_note = x)}
+                                        placeholder="Leave note"
+                                        id="moderator-note"
+                                    />
+                                    <button onClick={this.addModeratorNote}>Add note</button>
+                                </div>
+                                <PaginatedTable
+                                    className="moderator-log"
+                                    name="moderator-log"
+                                    source={`moderation?player_id=${this.user_id}`}
+                                    ref={this.moderator_log_table_ref}
+                                    columns={[
+                                        {
+                                            header: "",
+                                            className: "date",
+                                            render: (X) =>
+                                                moment(X.timestamp).format("YYYY-MM-DD HH:mm:ss"),
+                                        },
+                                        {
+                                            header: "",
+                                            className: "",
+                                            render: (X) => <Player user={X.moderator} />,
+                                        },
+                                        {
+                                            header: "",
+                                            className: "",
+                                            render: (X) => (
+                                                <div>
+                                                    <div className="action">
+                                                        {X.game ? (
+                                                            <Link to={`/game/${X.game.id}`}>
+                                                                {X.game.id}
+                                                            </Link>
+                                                        ) : null}
+                                                        {X.action}
+                                                    </div>
+                                                    {X.incident_report && (
+                                                        <div>
+                                                            {X.incident_report.cleared_by_user ? (
+                                                                <div>
+                                                                    <b>Cleared by user</b>
+                                                                </div>
+                                                            ) : null}
+                                                            <div>{X.incident_report.url}</div>
```

Comment:
This was one of the nastiest areas of the 80 column test run:

<img width="484" alt="Screen Shot 2022-01-15 at 7 44 14 AM" src="https://user-images.githubusercontent.com/25233703/149627960-d56f3cbb-6c5c-4935-8f86-1ab6a7dd9149.png">

Looks fine on 100 cols!